tasks/task1/task1_1.js

```diff
@@ -0,0 +1,8 @@
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('readable', function() {
+    let chunk;
+    while ((chunk = process.stdin.read()) !== null) {
+        process.stdout.write(`${chunk.split('').reverse().join('')}`);
```

Comment:
Consider storing this string in a separate constant to improve readability of your code.
Also, please add an end-of-line character at the end of it - to improve UX :)